apps/framework-cli/src/framework/core/infrastructure_map.rs

```diff
@@ -2183,7 +2220,8 @@ impl PartialInfrastructureMap {
             for transformation_target in &source_partial_topic.transformation_targets {
                 debug!("transformation_target: {:?}", transformation_target);
 
-                let process_id = format!("{}_{}", topic_name, transformation_target.name);
+                // In dmv1, the process name was the file name which had double underscores
+                let process_name = format!("{}__{}", topic_name, transformation_target.name);
```

Comment:
will this change cause problems when a user upgrades?

Reply:
It shouldn't. I made this change because there was a diff when I compared dmv1 vs dmv2, so I believe they would've had a problem without this change.